client/lib/plans/constants.js

```diff
@@ -519,3 +519,23 @@ export function isMonthly( plan ) {
 export function isPopular( plan ) {
 	return includes( POPULAR_PLANS, plan );
 }
+
```

Comment:
Not blocking, but:
isn't this better suited in `lib/plans/index` ?
Or perhaps a method / attribute on plans objects in constants.js? 
All plans properties are tied to plans constants objects now